src/components/accordion.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import { Collapse, Button, CardBody, Card } from 'reactstrap';
+import styled from "react-emotion";
+
+// CSS
+
+const XButton = styled(Button)`
```

Comment:
Changed all instances of "Styled" to "X" so `XButton` instead of `StyledButton`. What are your thoughts? Just seems shorter and we could make it a rule for all styled components. I really don't mind if you prefer a different character like an underscore for instance.  Just thought putting "Styled" so many times seemed a bit verbose.

Reply:
Could it just be `Button`?

Reply:
I wish, imports are usually constants so you can't overwrite them. `Button` is coming from reactsrap.

Reply:
![image](https://user-images.githubusercontent.com/9560798/62185855-fb284300-b331-11e9-8baf-5539769ad862.png)